core/main/src/Options/Interfaces/Particles/IParticles.ts

```diff
@@ -45,4 +46,5 @@ export interface IParticles {
     size: ISize;
     stroke: SingleOrMultiple<IStroke>;
     twinkle: ITwinkle;
+    orbit: IOrbit;
```

Comment:
Can you please sort it alphabetically? It's easier to update docs

Reply:
Sorted.

Reply:
I still see it under `twinkle`, but it should be under `opacity`. `@deprecated` properties are ignored in the sort, they won't be documented.